pkg/lib/cockpit.js

```diff
@@ -2277,9 +2277,15 @@ function factory() {
                 }
             });
 
-            iterate_data(file_content, function(data) {
-                replace_channel.send(data);
-            });
+            if (file_content !== null) {
+                if (file_content.length === 0 || file_content.byteLength === 0) {
```

Comment:
This feels special enough to warrant a small comment why this is an exception case?

Reply:
Sure.